xml/System.Runtime.CompilerServices/ConditionalWeakTable`2.xml

```diff
@@ -588,9 +597,20 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns an enumerator that can be used to iterate through the <see cref="T:System.Runtime.CompilerServices.ConditionalWeakTable`2" /> table.</summary>
+        <returns>An enumerator that can be used to iterate through the <see cref="T:System.Runtime.CompilerServices.ConditionalWeakTable`2" /> table.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This member is an explicit interface member implementation. It can be used only when the <xref:System.Runtime.CompilerServices.ConditionalWeakTable&lt;TKey,TValue&gt;> instance is cast to an <xref:System.Collections.IEnumerable> interface.
+
+The returned enumerator will not extend the lifetime of any object pairs in the table, other than the current one. It will not return entries that have already been collected, nor will it return entries added after the enumerator was
+retrieved. It may not return all entries that were present when the enumerator was retrieved, however, such as not returning entries that were collected or removed after the enumerator was retrieved but before they were enumerated.
```

Comment:
```suggestion
retrieved. Additionally, it may not return all entries that were present when the enumerator was retrieved, for example, entries that were collected or removed after the enumerator was retrieved but before they were enumerated.
```